### PR TITLE
GUI Improvements

### PR DESCRIPTION
**Pull request by VSEphpbb**
Added nicer badge like styling to status
Added nicer font icons to add/edit buttons
Added some post buttons to the idea (edit post, report post and post info)
Moved the idea's delete action to the quick mod drop down

screen shot 2017-05-14 at 10 28 18 am

### CODE REVIEW

File: language/en/common.php [hunk 1]
```diff
@@ -36,6 +37,7 @@
 	'IDEAS'					=> 'Ideas',
 	'IDEA_DELETED'			=> 'Idea successfully deleted.',
 	'IDEA_LIST'				=> 'Idea List',
+	'IDEA_NO'				=> 'Idea #',
```
Comment (DavidIQ) [inline language/en/common.php]: I think I'd rather this be `IDEA_NUM` as the current one makes it seem like it's for the idea being denied.

File: styles/prosilver/template/idea_body.html [hunk 2]
```diff
@@ -45,51 +59,44 @@
...
 					{% endif %}
 				</dd>
 			{% endif %}
 			{% if IDEA_DUPLICATE or S_IS_MOD %}
-				<dt class="duplicatetoggle" style="width: 15%; {% if IDEA_STATUS != lang('DUPLICATE') %}display:none{% endif %}">{{ lang('DUPLICATE') ~ lang('COLON') }}</dt>
-				<dd class="duplicatetoggle" {% if IDEA_STATUS != lang('DUPLICATE') %}style="display:none"{% endif %}>
-					<a id="duplicatelink" class="ideamodbtn" data-link="{{ U_IDEA_DUPLICATE }}" data-l-msg="{{ lang('VIEW_DUPLICATE') }}" {% if IDEA_DUPLICATE %}href="{{ U_IDEA_DUPLICATE }}">{{ lang('VIEW_DUPLICATE') }}{% else %}style="display:none">{% endif %}</a>
+				<dt class="duplicatetoggle idealabel"{% if IDEA_STATUS_ID != 4 %} style="display:none"{% endif %}>{{ lang('DUPLICATE') ~ lang('COLON') }}</dt>
+				<dd class="duplicatetoggle" {% if IDEA_STATUS_ID != 4 %}style="display:none"{% endif %}>
+					<a id="duplicatelink" class="ideamodbtn" data-link="{{ U_IDEA_DUPLICATE }}" data-l-msg="{{ lang('IDEA_NO') }}" {% if IDEA_DUPLICATE %}href="{{ U_IDEA_DUPLICATE }}">{{ lang('IDEA_NO') }}{{ IDEA_DUPLICATE }}{% else %}style="display:none">{% endif %}</a>
```
Comment (DavidIQ) [inline styles/prosilver/template/idea_body.html]: ~~`IDEA_NO`~~ => `IDEA_Num`

**Comment by VSEphpbb** [comment]
Updated

**Comment by VSEphpbb** [comment]
I have a couple more updates coming. Basically they involve legitimizing the Edit and Delete idea moderator actions...making the Edit of the first post work as normal, and moving the Delete Idea action into the Quick Mod menu

**Comment by VSEphpbb** [comment]
updated with an updated screenshot. Should be good for now.

**Comment by DavidIQ** [comment]
I know you removed the wip label, but you made additional changes after that. Are you really done? 🙂

**Comment by VSEphpbb** [comment]
@DavidIQ I'm never really done, but yes I'm done for now.

**Pull request merged**